archive.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Template Name: Archives
+ */
+get_header();
+?>
+
+<div class="container">
+	<div class="row">
+		<div class="col-md-8">
+			<?php today_archive_pagination(); ?>
+			<?php if ( have_posts() ) : ?>
+				<?php echo today_post_list_display_feature( '', $posts, array( 'posts_per_row' => 3, 'layout' => 'vertical' ) ); ?>
+				<?php ucfwp_the_posts_pagination(); ?>
+			<?php else : ?>
+				<div class="alert alert-info">
+					<p>There are currently no stories for <?php echo date( 'F Y' ); ?>.</p>
+				</div>
+			<?php endif; ?>
+		</div>
+		<div class="col-md-4">
+		<h2 class="h6 text-uppercase text-default-aw mb-4">Resources</h2>
+		<ul class="list-unstyled">
```

Comment:
We could use `wp_nav_menu` to output the Resources menu, it's used for the Tag template page:
https://github.com/UCF/Today-Child-Theme/blob/rc-v1.0.0/tag.php#L61-L66

Reply:
Added